package.json

```diff
@@ -6,12 +6,10 @@
   "author": "Kevin Barabash <kevinb7@gmail.com>",
   "license": "MIT",
   "dependencies": {
-    "@babel/eslint-parser": "^7.22.11",
     "@babel/traverse": "^7.22.11",
     "eslint": "^8.48.0",
+    "eslint-scope": "^7.2.2",
+    "eslint-visitor-keys": "^3.4.3",
     "swc-to-babel": "^2.2.0"
```

Comment:
We can get rid of this too since we're going convert from `swc` to `espree` using Rust.